weldx/tests/asdf_tests/test_asdf_validators.py

```diff
@@ -223,63 +223,7 @@ def test_shape_validator_exceptions(test_input):
 @pytest.mark.parametrize(
     "test",
     [
-        UnitValidatorTestClass(
-            length_prop=Q_(1, "inch"),
-            velocity_prop=Q_(2, "km / s"),
-            current_prop=Q_(np.eye(2, 2), "mA"),
-            nested_prop=dict(q1=Q_(np.eye(3, 3), "m"), q2=Q_(2, "m^3")),
-            simple_prop={"value": float(3), "unit": "m"},
-        ),
-        pytest.param(
-            UnitValidatorTestClass(
-                length_prop=Q_(1, "s"),  # wrong unit
-                velocity_prop=Q_(2, "km / s"),
-                current_prop=Q_(np.eye(2, 2), "mA"),
-                nested_prop=dict(q1=Q_(np.eye(3, 3), "m"), q2=Q_(2, "m^3")),
-                simple_prop={"value": float(3), "unit": "m"},
-            ),
-            marks=pytest.mark.xfail(raises=ValidationError),
-        ),
-        pytest.param(
-            UnitValidatorTestClass(
-                length_prop=Q_(1, "s"),
-                velocity_prop=Q_(2, "liter"),  # wrong unit
-                current_prop=Q_(np.eye(2, 2), "mA"),
-                nested_prop=dict(q1=Q_(np.eye(3, 3), "m"), q2=Q_(2, "m^3")),
-                simple_prop={"value": float(3), "unit": "m"},
-            ),
-            marks=pytest.mark.xfail(raises=ValidationError),
-        ),
-        pytest.param(
-            UnitValidatorTestClass(
-                length_prop=Q_(1, "inch"),
-                velocity_prop=Q_(2, "km / s"),
-                current_prop=Q_(np.eye(2, 2), "V"),  # wrong unit
-                nested_prop=dict(q1=Q_(np.eye(3, 3), "m"), q2=Q_(2, "m^3")),
-                simple_prop={"value": float(3), "unit": "m"},
-            ),
-            marks=pytest.mark.xfail(raises=ValidationError),
-        ),
-        pytest.param(
-            UnitValidatorTestClass(
-                length_prop=Q_(1, "m"),
-                velocity_prop=Q_(2, "km / s"),
-                current_prop=Q_(np.eye(2, 2), "mA"),
-                nested_prop=dict(q1=Q_(np.eye(3, 3), "m"), q2=Q_(2, "V")),  # wrong unit
-                simple_prop={"value": float(3), "unit": "m"},
-            ),
-            marks=pytest.mark.xfail(raises=ValidationError),
-        ),
-        pytest.param(
-            UnitValidatorTestClass(
-                length_prop=Q_(1, "m"),
-                velocity_prop=Q_(2, "km / s"),
-                current_prop=Q_(np.eye(2, 2), "mA"),
-                nested_prop=dict(q1=Q_(np.eye(3, 3), "m"), q2=Q_(2, "m^3")),
-                simple_prop={"value": float(3), "unit": "s"},  # wrong unit
-            ),
-            marks=pytest.mark.xfail(raises=ValidationError),
-        ),
+        UnitValidatorTestClass(),
     ],
 )
 def test_unit_validator(test):
```

Comment:
I think we can remove the parametrization here

Reply:
added another test to cover conversion just in case